test/accessibility/edu-1995-508.spec.js

```diff
@@ -0,0 +1,131 @@
+
+      // Service periods page.
+      client
+        .waitForElementVisible('label[for="root_view:newService"]', Timeouts.slow);
+      EduHelpers.completeServicePeriods(client, EduHelpers.testData.servicePeriods.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/military-history/service-periods');
+
+      // Military service page.
+      client
+        .waitForElementVisible('label[for="root_view:hasServiceBefore1978"]', Timeouts.slow);
+      // Another mysteriously required pause. If we don't wait here, then the click below will
+      // do nothing sometimes.
+      client.pause(1000);
+      EduHelpers.completeMilitaryService(client, EduHelpers.testData.militaryHistory.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/military-history/military-service');
+
+      // New school
+      client
+        .waitForElementVisible('label[for="root_educationType"]', Timeouts.slow);
+      EduHelpers.completeNewSchool(client, EduHelpers.testData.newSchool.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/school-selection/new-school');
+
+      // Old school
+      client
+        .waitForElementVisible('label[for="root_oldSchool_name"]', Timeouts.slow);
+      EduHelpers.completeOldSchool(client, EduHelpers.testData.oldSchool.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/school-selection/old-school');
+
+      // Contact information page.
+      client
+        .waitForElementVisible('label[for="root_preferredContactMethod"]', Timeouts.slow);
+      EduHelpers.completeContactInformation(client, EduHelpers.testData.contactInformation.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/contact-information');
+
+      // Dependents
+      client
+        .waitForElementVisible('label[for="root_serviceBefore1977_married"]', Timeouts.slow);
+      EduHelpers.completeDependents(client, EduHelpers.testData.dependents.data, false);
+      client
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/dependents');
+
+      // Direct deposit page
+
+      // No longer works with the updated direct deposit page
+      // client
+      //   .waitForElementVisible('label[for="root_bankAccountChange"]', Timeouts.slow);
+      // EduHelpers.completeDirectDeposit(client, EduHelpers.testData.directDeposit.data, false);
```

Comment:
For the record, @jbalboni recommended offline that I run `npm install` again. That cleared it up, so I uncommented those lines.